src/test/unit/test_tracer.py

```diff
@@ -364,3 +370,57 @@ def lambda_test_function(event, context):
     # following python's runtime: runtime/lambda_runtime_marshaller.py:27
     expected_message = 'The lambda will probably fail due to bad return value. Original message: "Object of type datetime is not JSON serializable"'
     assert function_span["error"]["message"] == expected_message
+
+
+@mock_kinesis
+def test_china(context, reporter_mock, monkeypatch):
+    china_region_for_test = "ap-east-1"  # Moto doesn't work for China
+    monkeypatch.setattr(lumigo_utils, "CHINA_REGION", china_region_for_test)
+    monkeypatch.setenv("AWS_REGION", china_region_for_test)
```

Comment:
IMO you don't really need these lines, as long as line 385 will use the fake region 🤔

Reply:
We need this because in `report_json` there is:
```
    if region == CHINA_REGION:
        return _publish_spans_to_kinesis(to_send, CHINA_REGION)
```
And the env variable will set the `region` variable.

When I tried to remove it the test failed because the tracer tried to send the spans via http (the old way) instead of pushing them to Kinesis

Reply:
yeah, but what if you'll just use the real china region?
the only reason not to do it is moto, 
and you're mocking it either way 🤷